torch/_guards.py

```diff
@@ -561,6 +561,21 @@ def current_trace_id():
         return TraceId(self.compile_id, self.attempt)
 
 
+class FakificationPolicy:
```

Comment:
I hacked in a fix to unblock myself wherein I make and store a policy to reuse later. We should probably just make this a detail of the fake tensor mode tho, instead of plumbing it around above it.